Initial implementation of EnaSolar integration

geustace opened the pull request:
## Proposed change
<!-- 
    This is the initial documentation for and integration to create sensors to monitor an EnaSolar Solar Inverter
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
   https://github.com/home-assistant/core/pull/54859
-->
- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54859
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2759
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

geustace committed:
Initial implementation of EnaSolar integration

geustace committed:
Update enasolar.markdown

Correct typo in the inverter URL

frenck commented on source/_integrations/enasolar.markdown:
````diff
@@ -0,0 +1,95 @@
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: enasolar
+    host: IP_ADDRESS_OF_DEVICE
+```
````
For integrations that connect to devices or services, we no longer accept new YAML configuration.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

frenck committed:
2021.9: Initial beta release notes

raman325 committed:
Add some notes around zwave_js (#19082)

bieniu committed:
Add information about Xiaomi Miio pedestal fans (#19068)

raman325 committed:
Add docs for template.number platform (#18975)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>
Co-authored-by: Franck Nijhof <git@frenck.dev>

raman325 committed:
Document template.select platform (#19062)

Co-authored-by: Franck Nijhof <git@frenck.dev>

raman325 committed:
Add missed references to template integration docs (#19091)

apaperclip committed:
Fix typo in lock optimistic variable description (#19089)

balloob committed:
Add link to flashing Blue directly (#19093)

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>

frenck committed:
2021.9: Prepare final release notes

frenck committed:
Sync codebase with docs for 2021.9

frenck committed:
2021.9: Fix typo/wording in other noteworthy section

frenck committed:
2021.9: Apostrophe abuse

frenck committed:
2021.9 Screenshot -> Animation

bdraco committed:
Remove zig-a-zig-ah from discovery tested devices (#19084)

Electrolama zig-a-zig-ah doesn't have any identifiers that allow us to keep it with the new design in https://github.com/home-assistant/core/pull/55236

geustace committed:
Update enasolar.markdown

Rewrite for config_flow instead of editing YAML directly

geustace committed:
Fix spelling

JonGilmore committed:
Update default Lutron username/password (#19121)

frenck committed:
2021.9 Minor typos

gtdiehl committed:
Add IoTaWatt integration doc (#18787)

balloob committed:
Update social

duncanvanzyl committed:
Update humidifier.mqtt.markdown (#19140)

Remove duplicate modes entry in configuration.

balloob committed:
Update blue price

balloob committed:
Mention price availability for Eagle integration (#19145)

emontnemery committed:
Clarify last_reset configuration for MQTT sensor (#19148)

milanmeu committed:
Remove OwnTracks with two MQTT brokers example (#19125)

ohadbenita committed:
Picture-elements markdown: Add `transform:none`. (#19147)

Co-authored-by: Joakim Sørensen <hi@ludeeus.dev>

klaasnicolaas committed:
Update P1 Monitor documentation (#19130)

bramkragten committed:
Update release 2021.9 notes for final

balloob committed:
Merge pull request #19161 from home-assistant/release-notes-2021.9

balloob committed:
Add documentation on how to use IoTaWatt with energy management (#19155)

* Add documentation on how to use IoTaWatt with energy management

* Update source/_integrations/iotawatt.markdown

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>

bramkragten committed:
Fix release notes (#19162)

DataBitz committed:
Legacy firmware 1.26 (#19164)

Tested use of legacy protocol with DSP-W215 and firmware 1.26

balloob committed:
2021.9.1

Petro31 committed:
Fix template number & select quoting errors (#19186)

bieniu committed:
Improve information about entities naming in Shelly integration (#19192)

dgomes committed:
Update cycle parameter in utility_meter (#19182)

jelmerkk committed:
Add gas usage graph to energy.markdown (#19190)

Co-authored-by: Joakim Sørensen <hi@ludeeus.dev>

ludeeus committed:
Add step to number template example (#19189)

balloob committed:
2021.9.2

chk1 committed:
Fix small formatting error in documentation (#19197)

geustace committed:
Merge branch 'home-assistant:current' into enasolar

gtdiehl commented:
@geustace Should probably add the link to the [brands PR](https://github.com/home-assistant/brands/pull/2759) in the beginning of the PR

ludeeus commented:
You merged `current` into your branch, but it's targetting `next`, which added a lot of unrelated commits to this PR.

geustace commented:
Merging current was not what I intended. Not sure how to unwind, not a Git expert.

gtdiehl commented:
@geustace You could close the PR and start over.

geustace commented:
I thought that was going to be the answer :-(

geustace commented:
Going to try again
